.clang-format

```diff
@@ -107,7 +107,7 @@ JavaScriptWrapImports: true
 KeepEmptyLinesAtTheStartOfBlocks: true
 MacroBlockBegin: ''
 MacroBlockEnd:   ''
-MaxEmptyLinesToKeep: 1
+MaxEmptyLinesToKeep: 2
```

Comment:
Is this changing the clang-format as well in this PR?

Reply:
This was in a different PR and it got closed. I don't think it actually changes any lines since we previously removed all gaps of 2 or more lines.  This just allows them to come back if you want to.

Reply:
Correct.  I intended that to be in the clang-format original, but somehow it got changed back and it would have been too painful to go back to the original code a reformat since there was a lot of changes I had to do by hand to make it compile after changing headers around.  This at least allows two spaces in the future to add a bit of separation between groups of functions, variables, etc.